src/index.mts

```diff
@@ -0,0 +1,140 @@
+import express, {type Application} from 'express';
+import cookieParser from 'cookie-parser';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import {FusionAuthSDK} from './sdk.ts';
+
+// ESM workaround for __dirname
+const __filename: string = fileURLToPath(import.meta.url);
+const __dirname: string = dirname(__filename);
+const port: number = 8080; // default port to listen
+
+const sdk: FusionAuthSDK = new FusionAuthSDK(
+    {
+      apiKey: 'noapikeyneeded',
+      applicationId: 'e9fdb985-9173-4e01-9d73-ac2d60d1dc8e',
+      baseURL: 'http://localhost:9011',
+      clientId: 'e9fdb985-9173-4e01-9d73-ac2d60d1dc8e',
+      clientSecret: 'super-secret-secret-that-should-be-regenerated-for-production',
+      port: port
+    }
+);
+
+const app: Application = express();
+
+app.use(cookieParser());
+app.use(express.urlencoded());
+
+// Static Files
+app.use('/static', express.static(path.join(__dirname, '../static/')));
+
+// Homepage (always display this)
+app.get("/", async (_req, res) => {
+  res.sendFile(path.join(__dirname, '../templates/home.html'));
+});
+
+// Login route
+app.get('/login', async (req, res) => {
+  if (await sdk.userLoggedIn(req, res)) {
+    res.redirect(302, '/account');
+    return;
+  }
+
+  sdk.sendToLoginPage(res);
```

Comment:
This is so cool. I hope we can merge this into fusionauth-typescript-client (or at least move it outside of this repo).